### PR TITLE
#1262 add Service to Worker pattern

### PR DESCRIPTION
**Pull request by zwebrain**
* Resolves issue [#1262](https://github.com/iluwatar/java-design-patterns/issues/1262)
* The idea of the pattern is for clients to use a controller to centralized control the dispatcher to handle requests and prepare a dynamic presentation. It can decouple navigation from the front controller by controlling the execution flow and access flow of business data.

### CODE REVIEW

File: service-to-worker/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+The system controls the execution flow and access flow of business data, creates presentation content from it, and centralizes control and improves modularity, reuse, and separation of roles.
+
+
+**Example**
+
+We modified this pattern based on a classic design patterns [Model View Controller Pattern](https://github.com/iluwatar/java-design-patterns/tree/master/model-view-controller) as the Class Diagram.
+
+In this example, the controller directly controls the display of the View and can receive commands to control the dispatcher indirectly. The dispatcher stores different commands that can be used to modify the `model` with `action`s or to modify the display in the `view`s.
+
+Therefore, this example leverages the Service to Worker pattern to increase functionality cohesion and improve the business logic.
```
Comment (iluwatar) [inline service-to-worker/README.md:24]: Please use the same explanation format as we do in other patterns e.g. https://github.com/iluwatar/java-design-patterns/tree/master/abstract-factory

File: service-to-worker/src/main/java/com/iluwatar/servicetoworker/Action.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
+package com.iluwatar.servicetoworker;
+
+import com.iluwatar.model.view.controller.Fatigue;
+import com.iluwatar.model.view.controller.Health;
+import com.iluwatar.model.view.controller.Nourishment;
+
+/**
+ * The type Action (Worker), which can processed user input and perform a specific update on the
```
Comment (iluwatar) [inline service-to-worker/src/main/java/com/iluwatar/servicetoworker/Action.java:8]:
```suggestion
 * The type Action (Worker), which can process user input and perform a specific update on the
```

File: service-to-worker/src/main/java/com/iluwatar/servicetoworker/Command.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    return nourishment;
+  }
+
+  Health getHealth() {
+    return health;
+  }
+
+  Fatigue getFatigue() {
+    return fatigue;
+  }
```
Comment (iluwatar) [inline service-to-worker/src/main/java/com/iluwatar/servicetoworker/Command.java:39]: It's possible to use Lombok getters for these

Comment (iluwatar) [reply service-to-worker/src/main/java/com/iluwatar/servicetoworker/Command.java:39]: Check elsewhere too to reduce boilerplate. Some guidelines available at https://github.com/iluwatar/java-design-patterns/wiki/18.-Lombok

**Review by iluwatar — CHANGES_REQUESTED** [review]

File: service-to-worker/src/main/java/com/iluwatar/servicetoworker/Command.java (hunk 3 above)
Comment (zWeBrain) [reply service-to-worker/src/main/java/com/iluwatar/servicetoworker/Command.java:39]: Thank you, I will continue to work on these.

**Comment by iluwatar** [comment]
Closed due to inactivity

**Pull request closed without merging**